### ISSUE TITLE
Any luck getting more data?

### ISSUE THREAD

**Issue by Stuhl** [XcomReborn/COH1-Python-Replay-Parser#1]
Like command per player per second etc.

**Comment by XcomReborn** [comment]
Not implemented no, however if you can read C# then 

https://www.gamereplays.org/community/index.php?showtopic=813678

pingtofts code was what I adapted into python, just to parse the header.

His code however does tick parsing too to find things like doctrines messages and actions.

I 'could' copy his methods in python BUT I really have all the information I need from the replay file header.

Feel free to fork this and have a go yourself.

**Comment by Stuhl** [comment]
Might look into it
Unfortunately I'm not experienced in reverse engineering stuff like this, any direction you can point me to for learning?

**Comment by XcomReborn** [comment]
Yes, get a byte viewer and look at the bytes in the coh replay files.

Then look at readings files byte by byte and casting those bytes into strings or ints
